pdfbox/src/main/java/org/apache/pdfbox/pdmodel/font/encoding/MacRomanEncoding.java

```diff
@@ -29,6 +29,7 @@ public class MacRomanEncoding extends Encoding
     /**
      * Table of octal character codes and their corresponding names.
      */
+    @SuppressWarnings("OctalInteger")
```

Comment:
same as above

Reply:
there are other SuppressWarnings (like "squid:S106" or "SpellCheckingInspection"). There are just a few warnings defined in Java, so that annotation mostly adresses 'other tools'.

Reply:
We are using sonar and I guess it was added to address that one

Reply:
Yes, either that one or another tool we were using.